Transport/Smtp.php

```diff
@@ -335,6 +317,11 @@ public function send ( \Hoa\Mail\Message $message ) {
     /**
      * H-MAC.
```

Comment:
oki I didn't remember this function so just what you just said ;) "a well-known cryptographic function" ;)